website/content/en/preview/AWS/provisioning.md

```diff
@@ -189,6 +189,49 @@ spec:
           snapshotID: snap-0123456789
 ```
 
+### UserData
+
+The `userData` field in a Provisioner can be used to control the UserData that Karpenter will use when provisioning nodes.
+Your UserData must be a Base64 encoded string, and it will be processed different based on the AMIFamily selected.
+
+*This field is currently only supported for the BottleRocket AMIFamily*.
+
+**For BottleRocket**
+* Your UserData must be valid TOML.

```

Comment:
oops, I think these got reverted when you changed the line breaks

Reply:
Nooooooo